docker-compose.yml

```diff
@@ -1,150 +1,126 @@
-version: "3"
+x-app: &app
+  build:
+    target: development
+    context: .
+    args:
+      RUBY_VERSION: '3.1.4'
+      PG_MAJOR: '11'
+      NODE_MAJOR: '16'
+      YARN_VERSION: '1.22.18'
+  image: ghcr.io/forem/forem:development
+  environment: &env
+    NODE_ENV: ${NODE_ENV:-development}
+    RAILS_ENV: ${RAILS_ENV:-development}
+  tmpfs:
+    - /tmp
+    - /app/tmp/pids
+
+x-backend: &backend
+  <<: *app
+  stdin_open: true
+  tty: true
+  volumes:
+    - .:/app:cached
+    - bundle:/usr/local/bundle
+    - rails_cache:/app/tmp/cache
+    - assets:/app/public/assets
+    - node_modules:/app/node_modules
+    - packs:/app/public/packs
+    - packs-test:/app/public/packs-test
+    - history:/usr/local/hist
+    - ./.dockerdev/.psqlrc:/root/.psqlrc:ro
+    - ./.dockerdev/.bashrc:/root/.bashrc:ro
+  environment: &backend_environment
+    <<: *env
+    REDIS_URL: redis://redis:6379/
+    DATABASE_URL: postgres://postgres:postgres@postgres:5432
+    WEBPACKER_DEV_SERVER_HOST: webpacker
+    MALLOC_ARENA_MAX: 2
+    WEB_CONCURRENCY: ${WEB_CONCURRENCY:-1}
+    BOOTSNAP_CACHE_DIR: /usr/local/bundle/_bootsnap
+    XDG_DATA_HOME: /app/tmp/cache
+    YARN_CACHE_FOLDER: /app/node_modules/.yarn-cache
+    HISTFILE: /usr/local/hist/.bash_history
+    PSQL_HISTFILE: /usr/local/hist/.psql_history
+    IRB_HISTFILE: /usr/local/hist/.irb_history
+    EDITOR: vi
```

Comment:
IMO `vi` is not a user-friendly default here. Whether we use `nano`, [`micro`](https://micro-editor.github.io/), or something else entirely, is kinda up for debate, but `vi` is a brutal setting for `$EDITOR` - even vim/neovim would be a better bet. (I don't daily-drive Debian-based systems so it's possible that `vi` becomes a symlink to vim on Debian, but then let's not rely on it in `EDITOR`, and instead explicitly call vim... if we want vim as the git commit message editor and etc. all the other implications of setting `$EDITOR`)

Reply:
Ok removing this default